fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraTimeMapImpl.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.kernel.modeshape;
+
+import java.util.Optional;
+
+import javax.jcr.Node;
+import javax.jcr.PathNotFoundException;
+import javax.jcr.RepositoryException;
+
+import org.fcrepo.kernel.api.exception.PathNotFoundRuntimeException;
+import org.fcrepo.kernel.api.exception.RepositoryRuntimeException;
+import org.fcrepo.kernel.api.models.FedoraResource;
+import org.fcrepo.kernel.api.models.FedoraTimeMap;
+
+import static org.fcrepo.kernel.modeshape.identifiers.NodeResourceConverter.nodeConverter;
+
+/**
+ * @author lsitu
+ * @since Oct. 04, 2017
+ */
+public class FedoraTimeMapImpl extends FedoraResourceImpl implements FedoraTimeMap {
+
+
+    /**
+     * Construct a {@link org.fcrepo.kernel.api.models.FedoraResource} from an existing JCR Node
+     * @param node an existing JCR node to be treated as a fcrepo object
+     */
+    public FedoraTimeMapImpl(final Node node) {
+        super(node);
+    }
+
+
+    @Override
+    public FedoraResource getTimeMap() {
```

Comment:
Now I'm not sure if this makes sense outside the FedoraResourceImpl. Calling `getTimeMap()` on the LDPRv resource to return the TimeMap made sense. Now we are calling `getTimeMap()` on the TimeMap...unsure.

Reply:
@whikloj Yep, it's not as easy understanding as adding it FedoraResourceImpl. But the FedoraTimeMap model can be interpreted by the FedoraResourceConverter as its own. This could be addressed later when we use the fcr:version endpoint for versioning. 
Would you like me to add another commit to address it now?

Reply:
No you know what its fine. I think some of this is just my own trouble fully understanding.